src/20241213_AaveV3Ethereum_AddFBTCToAaveV3MainMarketOnEthereum/AddFBTCToAaveV3MainMarketOnEthereum.md

```diff
@@ -0,0 +1,73 @@
+
+This new asset will broaden opportunities for Bitcoin holders aiming to participate in DeFi on Aave v3. The introduction of FBTC offers users more flexibility in leveraging their Bitcoin, enhancing liquidity and driving increased engagement within the Aave protocol.
+
+With $480M in total value locked (TVL), FBTC is emerging as a compelling solution in Bitcoin bridging, supported by its upcoming Chainlink Oracle integration to accelerate growth across EVM networks.
+
+Aave is positioned to benefit from a material increase in AUM resulting from FBTC deposits. This capital is being sourced by from within the Ignition team’s network. Several sophisticated investors are looking to user Aave at size and some teams have built products specifical for investors in anticipationg of the Aave listing. The Ignition team is expected to provide incentives supporting Aave users directly and also strategy providers built on top of Aave. @TokenLogic has been coordinating with various prospective users to ensure there is adequate demand ahead of the listing.
+
+## Specification
+
+### Listing
+
+The table below illustrates the configured risk parameters for **FBTC**
+
+| Parameter                 |                                      Value |
+| ------------------------- | -----------------------------------------: |
+| Isolation Mode            |                                      false |
+| Borrowable                |                                    ENABLED |
+| Collateral Enabled        |                                       true |
+| Supply Cap (FBTC)         |                                        200 |
+| Borrow Cap (FBTC)         |                                        100 |
+| Debt Ceiling              |                                      USD 0 |
+| LTV                       |                                       73 % |
+| LT                        |                                       78 % |
+| Liquidation Bonus         |                                      7.5 % |
+| Liquidation Protocol Fee  |                                       10 % |
+| Reserve Factor            |                                       50 % |
+| Base Variable Borrow Rate |                                        0 % |
+| Variable Slope 1          |                                        4 % |
+| Variable Slope 2          |                                      300 % |
+| Uoptimal                  |                                       45 % |
+| Flashloanable             |                                    ENABLED |
+| Siloed Borrowing          |                                   DISABLED |
+| Borrowable in Isolation   |                                   DISABLED |
+| Oracle                    | 0xF4030086522a5bEEa4988F8cA5B36dbC97BeE88c |
```

Comment:
Would be good to add also the current latestAnswer of the oracle (with a date) on this table or below it.